### PR TITLE
Password Vulnerability Test

### LINKED ISSUE #993: Password Checker
# Description

Describe the new feature you are proposing.

| name            | possible solution  | assignees                                                      |
| --------------- |------- | -------------------------------------------------------------- |
| Password Checker | This script uses API of https://haveibeenpwned.com/ and checks how many times your password has been hacked | @RohiniRG  |

I am a GSSOC21 Participant @HarshCasper kindly assign me the same

### PR DESCRIPTION
**Pull request by RohiniRG**
# Description

The python script checks if a password has been exposed to a data breach and the number of time it has occured, thus proving the password to be vulnerable for further attacks.

Fixes #993 

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update
- [x] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by RohiniRG** [comment]
@vybhav72954 could you please have a look

File: Python/Pwned_Passwords/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Running the script:
+```shell
+$ python pwned_checker.py
+```
+
+## Working screenshots
+![Image](https://i.imgur.com/3SekSwP.png)
+
+## References:
+https://haveibeenpwned.com/API/v3
```
Comment (vybhav72954) [inline Python/Pwned_Passwords/README.md:23]: A hyperlink, please in place of a direct URL

File: Python/Pwned_Passwords/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Password Vulnerability Checker
+
+- The python script uses haveibeenpwned.com API to check if your password has been exposed to data breaches and how many times it has been seen.
```
Comment (vybhav72954) [inline Python/Pwned_Passwords/README.md:3]: haveibeenpwned.com in backticks, similary. pwned in bold

File: Python/Pwned_Passwords/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Password Vulnerability Checker
```
Comment (vybhav72954) [inline Python/Pwned_Passwords/README.md:1]: More like a test rather than a check?

**Review by vybhav72954 — CHANGES_REQUESTED** [review]

**Comment by RohiniRG** [comment]
@vybhav72954 I made changes to the README accordingly, please have a look

File: Python/Pwned_Passwords/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+- The python script uses ```haveibeenpwned.com``` API to check if your password has been exposed to data breaches and how many times it has been seen.
+- Such a <b>pwned</b> password should no longer be used as its exposure puts it at higher risk of being used to login to accounts using the now-exposed secret.
+
+## Setting up
+
+- For running the script, the packages can be installed as follows
+```shell
+$ pip install -r requirements.txt
+```
+- Paste the token (password) in a .env file (Take the .env.example as an example)
```
Comment (vybhav72954) [inline Python/Pwned_Passwords/README.md:12]: Revert these changes please. ```.env``` looked good.

**Review by vybhav72954 — CHANGES_REQUESTED** [review]

**Comment by RohiniRG** [comment]
@vybhav72954 I reverted the few changes in README

**Comment by RohiniRG** [comment]
@vybhav72954 @HarshCasper could you please have a look

**Review by vybhav72954 — APPROVED** [review]
Thank you for a well structured PR

**Pull request merged**